### PR TITLE
feat/styles(Compare): Code Compare - Tabs and Settings

### PR DESCRIPTION
**Pull request by luvkapur**
Screenshot 2023-01-11 at 7 01 18 PM

Screenshot 2023-01-11 at 7 03 03 PM

Screenshot 2023-01-11 at 7 02 10 PM

### CODE REVIEW

File: components/ui/component-compare/component-compare/component-compare.module.scss [hunk 1]
```diff
@@ -16,15 +16,16 @@
...
-  padding-bottom: 16px;
+  // padding-bottom: 16px;
 }
 
 .navContainer {
   display: flex;
   align-items: center;
   min-height: 48px;
-  border-bottom: 1px solid var(--bit-border-color-lightest, #ededed);
+  border-bottom: 1px solid var(--bit-border-color-lightest, hsl(0, 0%, 93%));
```
Comment (odedre) [inline components/ui/component-compare/component-compare/component-compare.module.scss:26]: are you sure this is working? `-bit-border-color-lightest` should work here

File: scopes/code/ui/code-compare/code-compare-tree/code-compare-tree.module.scss [hunk 2]
```diff
@@ -30,3 +37,12 @@
   font-weight: unset;
   padding: 4px;
 }
+
+.node {
+  &:hover {
+    background-color: var(--surface-primary-hover-color, #3a2980) !important;
```
Comment (odedre) [inline scopes/code/ui/code-compare/code-compare-tree/code-compare-tree.module.scss:43]: can we add a comment why we need important ?

File: scopes/code/ui/code-compare/code-compare-view/code-compare-view.module.scss [hunk 3]
```diff
@@ -24,7 +28,7 @@
 }
 
 .ignoreWhitespaceControlContainer {
-  color: var(--bit-text-color, #414141);
+  color: var(--bit-text-color, var(--on-surface-neutral-low-color, #282828));
```
Comment (odedre) [inline scopes/code/ui/code-compare/code-compare-view/code-compare-view.module.scss:31]: fix

File: scopes/code/ui/code-compare/code-compare-view/code-compare-view.module.scss [hunk 4]
```diff
@@ -44,8 +48,156 @@
...
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+
+  > * {
+    margin-right: 4px;
+  }
+  > :last-child() {
+    margin-right: 0px;
+  }
```
Comment (odedre) [inline scopes/code/ui/code-compare/code-compare-view/code-compare-view.module.scss:202]: consider splitting the tabs css to a separate file

File: scopes/code/ui/code-compare/code-compare-view/code-compare-view.tsx [hunk 5]
```diff
@@ -40,18 +65,35 @@ export function CodeCompareView({ className, fileName }: CodeCompareViewProps) {
...
     loadingFromContext || !!codeCompareDataForFile?.compareContent
   );
+  const { fileContent: downloadedBaseFileContent, loading: loadingDownloadedBaseFileContent } = useFileContent(
+    componentCompareContext?.base?.model.id,
+    fileName,
+    loadingFromContext || !!codeCompareDataForFile?.baseContent
+  );
 
-  const loading = loadingFromContext || loadingDownloadedCompareFileContent || componentCompareContext?.loading;
+  const loading =
```
Comment (odedre) [inline scopes/code/ui/code-compare/code-compare-view/code-compare-view.tsx:90]: consider renaming `loadingAll` . not sure

File: scopes/code/ui/code-compare/code-compare.module.scss [hunk 6]
```diff
@@ -48,5 +63,32 @@ $panelBg: #fafafa;
...
 
 .loader {
-  color: var(--bit-bg-dent, #f6f6f6);
+  // @todo get token for this color
+  background: var(--on-surface-neutral-low-color, #282828);
+}
+
+.codeCompareTreeCollapse {
+  padding: 8px;
+  // @todo get token for this color
```
Comment (odedre) [inline scopes/code/ui/code-compare/code-compare.module.scss:72]: cleanup

File: scopes/component/component/ui/menu/nav-plugin.tsx [hunk 7]
```diff
@@ -6,6 +6,7 @@ import { ComponentModel } from '../../ui';
 
 export type NavPluginProps = {
   displayName?: string;
+  ignoreStickyQueryParams?: boolean;
```
Comment (odedre) [inline scopes/component/component/ui/menu/nav-plugin.tsx:9]: ignoreQueryParams

File: workspace.jsonc [hunk 8]
```diff
@@ -88,16 +88,17 @@
         "@teambit/defender.content.linter-overview": "1.95.0",
         "@teambit/defender.content.tester-overview": "1.95.0",
         "@teambit/design.elements.icon": "1.0.5",
-        "@teambit/design.inputs.dropdown": "0.0.7",
         "@teambit/design.inputs.selectors.multi-select": "0.0.20",
         "@teambit/design.inputs.toggle-switch": "0.0.6",
         "@teambit/design.navigation.content-tabs": "0.0.5",
-        "@teambit/design.navigation.responsive-navbar": "0.0.2",
+        "@teambit/design.themes.dark-theme": "1.91.4",
+        "@teambit/design.themes.light-theme": "0.1.1",
```
Comment (odedre) [inline workspace.jsonc:95]: I think we can remove the light theme

**Review by odedre — DISMISSED** [review]

**Review by davidfirst — APPROVED** [review]

**Pull request merged**